src/core/physics/CollisionInfo.js

```diff
@@ -0,0 +1,70 @@
+
+define(["lib/classy",
+        "lib/three"
+    ], function(Class, THREE) {
+
+/**
+    Info object that is returned when collision happens between two rigidbodies.
+    @class CollisionInfo
+    @constructor
+    
+    @param {Entity|Null} bodyA
+    @param {Entity|Null} bodyB
+    @param {THREE.Vector3} position
+    @param {THREE.Vector3} normal
+    @param {Number} distance
+    @param {Number} impulse
+    @param {boolean} newCollision
+*/
+var CollisionInfo = Class.$extend(
+{
+    __init__ : function()
+    {
+        this.bodyA = null;
+        this.bodyB = null;
+        this.position = new THREE.Vector3();
+        this.normal = new THREE.Vector3();
```

Comment:
You can document these inline in the ctor. Both js doc engines we have used supports this. With the current jsdoc it would be.

```
/**
    Desc of the thing.
    @var {Three.Vector}
*/
this.normal = new THREE.Vector3();
```

We can go thought the docs closer to the release, once @cvetan5 pulls in the new grunt stuff it will also update to the new documentation tool. I can help out there.